systemtest/src/test/java/io/strimzi/systemtest/upgrade/ZookeeperUpgradeST.java

```diff
@@ -98,11 +100,19 @@ void testKafkaClusterDowngradeToOlderMessageFormat(ExtensionContext testContext)
             initLogMsgFormat = tmp.toString();
         }
 
+        String interBrokerProtocol = sortedVersions.get(0).messageVersion();
```

Comment:
This looks wrong. Is `messageVersion()` really the IBP, or is it the `log.message.format.version`?